WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/CollapsibleView.swift

```diff
@@ -46,6 +49,7 @@ struct CollapsibleView<Label: View, Content: View>: View {
                     Spacer()
                     if isCollapsible {
                         Image(uiImage: isCollapsed ? .chevronDownImage : .chevronUpImage)
+                            .foregroundColor(hasSubtleChevron ? Color(.textSubtle) : .black)
```

Comment:
From the design w3JBReW1UJesCf02sj43kR-fi-982_5822, it looks like the chevron is `accent` color for the parent view? If `black` is the intended color, using `text` color would look better in dark mode.

<img width="398" alt="Screenshot 2023-11-28 at 9 29 53 AM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/5ee07625-5402-4f74-93ca-542e29bf3dba">

```suggestion
                            .foregroundColor(hasSubtleChevron ? Color(.textSubtle) : .accent)
```

Reply:
Good catch! I missed that change to `accent` here.